### PR TITLE
feat: update global settings in database

### PR DESCRIPTION
**Pull request by Jimmba**
Get settings from database (if there aren't - create it) and apply. 
Put changes of settings in database.

### CODE REVIEW

File: src/js/main page/GlobalSettings.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            switcher.addEventListener('click', (event) => this.changeTheme(event, 'theme'));
+    }
+
+    changeSlider(slider) {
+        const difficult = slider.value;
+        this.settingsFromDB.optional.settingsProfile.difficult = difficult;
+        this.updateSettingsInDb();
+    }
+
+    renderDifficultSlider() {
```
Comment (mill9r) [inline src/js/main page/GlobalSettings.js:80]: Strange naming.

File: src/js/main page/GlobalSettings.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            body, 
+            null, 
+            null,
+            null, 
+            ['src' , 'https://cdn.jsdelivr.net/gh/gitbrent/bootstrap4-toggle@3.6.1/js/bootstrap4-toggle.min.js']
+        );
+    }
+
+    async init () {
+        this.api = new Api(BASE_HEROKU);
```
Comment (mill9r) [inline src/js/main page/GlobalSettings.js:157]: Is it possible to pass in init method already instantiated objects?

File: src/js/main page/GlobalSettings.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        }
+    }
+
+    loadScriptForToggler() {
+        addElement('script', 
+            body, 
+            null, 
+            null,
+            null, 
+            ['src' , 'https://cdn.jsdelivr.net/gh/gitbrent/bootstrap4-toggle@3.6.1/js/bootstrap4-toggle.min.js']
```
Comment (mill9r) [inline src/js/main page/GlobalSettings.js:152]: Does ti make sense to create constant variable?

File: src/js/main page/GlobalSettings.js (hunk 1 above)
Comment (Jimmba) [reply src/js/main page/GlobalSettings.js:80]: Renamed

File: src/js/main page/GlobalSettings.js (hunk 3 above)
Comment (Jimmba) [reply src/js/main page/GlobalSettings.js:152]: fixed

File: src/js/main page/GlobalSettings.js (hunk 2 above)
Comment (Jimmba) [reply src/js/main page/GlobalSettings.js:157]: fixed

**Pull request merged**